implementations/elixir/ockam/ockam_vault_software/test/vault_software_test.exs

```diff
@@ -14,6 +14,13 @@ defmodule Ockam.Vault.Software.Tests do
     end
   end
```

Comment:
@mikelodder7 -- do you mind adding a copy of this test and replacing the line `{:ok, handle} = SoftwareVault.default_init()` with `{:ok, handle} = SoftwareVault.file_init("/tmp")`

to see that it also produces the same hash? just to verify that the two vaults have consistent behavior.

Reply:
sure